src/Compilers/CSharp/Test/Semantic/Semantics/DelegateTypeTests.cs

```diff
@@ -11494,5 +11495,725 @@ .maxstack  0
   IL_0000:  ret
 }");
         }
+
+        internal static void CheckNames(IEnumerable<MetadataReader> readers, IEnumerable<StringHandle> handles, params string[] expectedNames)
```

Comment:
I was expecting this method and all references to be removed.

Reply:
Missed those - sorry